### ISSUE TITLE
apktool d + b VLC-Android-1.7.2-ARMv7.apk fails 

### ISSUE THREAD

**Issue by MarkusML** [iBotPeaches/Apktool#1108]
Dear apktool team,

if possible, please help me to fix the following problem:

apktool 2.0.2 failes to de+re-compile VLC-Android-1.7.2-ARMv7.apk

With a previous APK (VLC-Android-1.3.2-ARMv7.apk)
the problem did not exist.

Many thanks.

Details:

*** Software versions:

apktool: 2.0.2 (2.0.1 affected too)
Java: openjdk 1.8.0_65-b17
Kernel: 3.16.7-29-desktop (if of any interest)

*** How to reproduce?

wget http://get.videolan.org/vlc-android/1.7.2/VLC-Android-1.7.2-ARMv7.apk
apktool d VLC-Android-1.7.2-ARMv7.apk
apktool b -o VLC-Android-1.7.2-ARMv7-new.apk VLC-Android-1.7.2-ARMv7

*** Resulting error message:
```
styles.xml:5: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/WindowTitle'.
styles.xml:6: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/WindowTitleBackground'.
Exception in thread "main" brut.androlib.AndrolibException: brut.androlib.AndrolibException: brut.common.BrutException: could not exec command: [/tmp/brut_util_Jar_8527551507135450245.tmp, p, --forced-package-id, 127, --min-sdk-version, 7, --target-sdk-version, 22, --version-code, 1070204, --version-name, 1.7.2, -F, /tmp/APKTOOL8634808352219872308.tmp, -0, resources.arsc, -0, arsc, -I, /home/titan/apktool/framework/1.apk, -S, /tools/apktool/VLC-Android-1.7.2-ARMv7/res, -M, /tools/apktool/VLC-Android-1.7.2-ARMv7/AndroidManifest.xml]
        at brut.androlib.Androlib.buildResourcesFull(Androlib.java:465)
        at brut.androlib.Androlib.buildResources(Androlib.java:403)
        at brut.androlib.Androlib.build(Androlib.java:291)
(...)
```

**Comment by MarkusML** [comment]
Dear apktool team,

I solved the problem. It was caused by an old

   ~/apktool/framework/1.apk

file. When removing this file apktool automatically
creates a new one (size modification-date file-name):

  2744669 Sep  7 23:25 1.apk.old
  2288202 Dec 17 11:01 1.apk

And with the new version of 1.apk the above
mentioned error does not appear anymore.
So the command

apktool b -o VLC-Android-1.7.2-ARMv7-new.apk VLC-Android-1.7.2-ARMv7

runs without error and creates the new apk.

Therefore most likely this ticket can be closed.
If at all the ticket is merely a feature request,
i.e. apktool should update 1.apk properly
or should warn about a too old 1.apk.

Anyway many thanks for this great tool
which is very helpful in so many situations.

Best regards,
Markus
